napalm/base/mock.py

```diff
@@ -68,7 +68,7 @@ def mocked_data(path, name, count):
     filename = "{}.{}".format(os.path.join(path, name), count)
     try:
         with open(filename) as f:
-            result = json.loads(f.read())
+            result = ast.literal_eval(f.read())
```

Comment:
Hey @brandomando - only now looking at your PR (sorry!). Can you explain why do you think this change is necessary? That'd be a breaking change for many users that have the mock data in JSON format, e.g.,

```python
>>> txt = '{"test": null}'
>>> json.loads(txt)
{'test': None}
>>> ast.literal_eval(txt)
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/usr/lib/python3.6/ast.py", line 85, in literal_eval
    return _convert(node_or_string)
  File "/usr/lib/python3.6/ast.py", line 66, in _convert
    in zip(node.keys, node.values))
  File "/usr/lib/python3.6/ast.py", line 65, in <genexpr>
    return dict((_convert(k), _convert(v)) for k, v
  File "/usr/lib/python3.6/ast.py", line 84, in _convert
    raise ValueError('malformed node or string: ' + repr(node))
ValueError: malformed node or string: <_ast.Name object at 0x7f426efe8518>
>>> 
```